lib/ejs.js

```diff
@@ -601,11 +601,12 @@ Template.prototype = {
     var matches = this.parseTemplateText();
 
     if (matches && matches.length) {
-      matches.forEach(function (line, index) {
+      matches.forEach(function (line) {
         self.scanLine(line);
       });
-      if (this.isInTag)
+      if (this.isInTag) {
```

Comment:
👍

Reply:
In my experience, omitted brackets are a major source of confusion during refactoring.